### DISCUSSION TITLE
Would you be willing to post some of the use cases/prompts you've been using?

### DISCUSSION THREAD

**Discussion by dvansoye** [brianpetro/obsidian-smart-connections#164] [Q&A] [answered]
I just got the Smart Connections plugin working. I copied over my concept notes which is a fraction of my vault. So, far, its working and I'm super impressed.

Here's where I could use your help or rather, here's were we could all help each other and Brian as well.

A. What are your use cases?
B. What prompts have you found helpful?

I'll go first.
1. I've put together a list of a dozen questions. I trying to get Smart Connections/OpenAI to take what I have to the next level. 

Here is a link to the article that started me down this road. Pretty interesting at least to me. I think it could be useful for most people.

https://zettelkasten.de/posts/feynmans-darlings-become-brilliant/

Here are my 12 questions:
1. How can I wake up every morning feeling great and ready to face the day?  
2. How can I improve my healthspan? 
3. How can I further accelerate the learning process? For example, how can I better leverage AI in order to accelerate the learning process? 
4. How can I improve my ability to capture metrics on the learning process? I want to know how much time I spend on each book. I want to know how much time I spend reading articles and books on a particular subject/genre. 
5. How can I improve the quality of the information I consume? By improving the quality of the information I consume, I will be getting a greater return on my investment. 
6. How can I stage my learning journey better? I want to understand the field first before digging into the details. 
7. How can I radically improve my writing? 
8. How can I be a better mentor? For example, how can I be a better listener? What strategies can I use to help young people develop their skills and knowledge? Find books and articles on the topic of mentorship. How can I become better at Transformational Leadership (Bing Chat)? 
9. How can I connect with other coaches and experts on accelerated learning? What do I bring to the table? How can I make an offer that's irresistible? 
10. How can I improve memory? I proved on a recent trip that I could memorize long list of very detailed information using a modified version of the memory palace. What else can I do to progress? 
11. How I can improve my ability to write AI prompts that will allow me to work with generative AI to build a better list of questions and also better answer those questions?

These questions all fit the following criteria: 

1. They are related to personal growth
2. They are open-ended and exploratory
3. They are focused on improving some aspect of performance or quality
4. They are motivated by intrinsic factors such as curiosity, interest and challenge.

Note: Yes, I know that I only have 11 questions. GPT-3 actually generated the 4 points above when I asked it to generate questions about the 12 (er, 11) questions and then answer them.

B. Here's my prompt to Smart Connections (and the response):

Critique my 12 questions. What's missing from them?

Based on your notes, your 12 questions are focused on personal growth and improvement, and they are open-ended and exploratory. However, they seem to be heavily focused on individual skills and knowledge, and may not address broader issues or societal concerns. Additionally, some of the questions may benefit from being more specific and actionable, rather than broad and abstract. 

To improve your questions, you may want to consider adding questions that address broader societal issues, such as environmental sustainability, social justice, or economic inequality. You may also want to consider adding questions that are more specific and actionable, such as "What specific steps can I take to improve my public speaking skills?" or "What resources can I use to learn a new language fluently within a year?" 

Overall, your questions are a good starting point for personal growth and exploration, but adding more specificity and addressing broader societal concerns may help to make them more well-rounded and impactful.

//end

So, in summary, I'm looking for OpenAI/Smart Connections to help me "Transcend" to entirely new areas of research. Maybe this is too much to ask.

Any suggestions? Anyone else working on a hard problem using their Obsidian Vault and Smart Connections?

**Comment by brianpetro** [comment]
@dvansoye this is sooo helpful! I've been wanting to put out more content about workflows.

In recent issue https://github.com/brianpetro/obsidian-smart-connections/issues/159#issuecomment-1499908844, I briefly shared a workflow about how I'm using Dataview to build "alignment notes":
> Right now I'm including those things in what I'm calling an "alignment note" that uses Dataview to aggregate things like top priorities, objectives, etc. Then I add the link to the Smart Chat user input to include that note as context.

As for prompts, one of my most frequent is "Summarize my notes about..."

And something I'm working on that I think will be huge, is the ability to build your own "prompt chains" that could be called by command or via the Smart Chat. I imagine then we will be able to use Obsidian Canvases as shareable workflows. 

Thanks again for starting this thread! It will be incredibly helpful for people picking up Smart Connections for the first time and those who haven't quite figured it out, too!

Brian 🌴😊

**Comment by SeelingSeeling** [comment]
I love to connect different notes and syntesize new ideas out of them: 

Connect [[Note1]] with [[Note2]] and merge all my ideas, concepts, insights, thoughts, and perspectives into a groundbreaking, unprecedented, holistic, ultimate, and singular perspective. Do not mention individual perspectives, do not list individual perspectives, only speak from the new unified, coherent perspective that you have created. Clearly, explicitly, concretely, and comprehensibly explain this new perspective in three paragraphs. Write a coherent explanation that brings everything together and efficiently summarizes it.

**Reply by brianpetro** [reply]
@SeelingSeeling that's a beautiful example! 👏

I can't wait to try it for myself.

Thanks for sharing,
Brian 🌴

**Reply by dvansoye** [reply]
Wow! @SeelingSeeling this is excellent! Beyond simply the prompt, it would be interesting to know what sorts of things you've learned as a result of this prompt. Has it provided any insights? What sort of notes do you typically merge together?

**Reply by SeelingSeeling** [reply]
> Wow! @SeelingSeeling this is excellent! Beyond simply the prompt, it would be interesting to know what sorts of things you've learned as a result of this prompt. Has it provided any insights? What sort of notes do you typically merge together?

I love to connect my notes about "system theory" with anything I am curious about. This helps me to create a holistic understanding of everything i want to learn. For example: Music as a system, clouds as a system, the human psyche as a system etc. 
Then my abstract ideas about processes, emergence, hierarchy or fractals are getting connected to something real and concrete. 
For example, i learned that music is made out of simple parts (notes and rhythms) that are hierarchical ordered which are creating complex emergent patterns on different levels. The same thing applys for clouds: Clouds are made of simple parts (water particles) which are creating complex emergent patterns on different levels. 

I think it works best for interdisciplinary concepts.

**Reply by dvansoye** [reply]
Very cool. Thank you for sharing your unique perspective. So when you're using smart connections, are you asking it to combine two notes together or sometimes are you asking it to combine concepts together?

I'm wondering if you ever had some terrific insight as a result of that experimentation?

Also, Are you by chance using obsidian for your diary? I know I am. It would be interesting to run some searches against it to see if there are some common themes that are coming up over and over again.

**Reply by SeelingSeeling** [reply, marked as answer]
@dvansoye  Yes, I love it. 
A few days ago, I learned so much about resonance and harmony that it made my head spin, and I had to sleep for a while. 

If you want to know the insights concretely here they are, but I warn you, there are more on the esoteric side: 

I learned that resonance is amplification through alignment. This happens in systems, music, personal relationships, metaphysics and spirituality. 
I realized that what I want and need the most is to get into resonance with the things I most highly value: truth, love, beauty and god. I achieve this by aligning my thoughts, emotions and actions with those concepts through sheer will, desire and intention.
This is an emotional process which require me to deal mainly with suppressed feelings and memories which are restricting the unfolding of the alignment.

This is the basic overview of the insight, it gets a lot deeper.

About your other question: Yes, I journal with obsidian! I excluded my Journal Notes from smart connections because I didn't want it to clog up my conceptual ideas. Maybe I should try it. When I do, I will report back.

**Reply by dvansoye** [reply]
Definitely. I share a lot in my diary and I'm wondering what I can learn about myself. It would be like a Wise Advocate providing insight. In fact, I would create different personas and ask each of them for insight on a particular theme.

**Comment by srijrao** [comment]
I excluded all but one folder called "AI Brain" and just put notes that I want to ask questions about. I found that my notes were not organized enough to use it other ways. 
The main issue before this singular folder was that I use obsidian to keep logs of work activities, and those logs confuse the model. Also, everything I take notes on is simple enough that I don't find much use in finding things with the chat function.

**Comment by a2jc4life** [comment]
I'm new to Smart Connections and still waiting on it to load up my notes. But I've been playing around in ChatGPT with notes I copy/paste into it, so I think what I've been doing should carry over to native use with the plugin.

I started with a prompt I found in the Obsidian forum that asks it to generate atomic note ideas from a text you provide. (This is helpful, in my experience, when you have a vague sense that your note contains more than one idea but are having a hard time sussing them out.) But then I figured out that I could piggyback them: I could keep feeding it more notes on a similar topic and it would keep spitting out atomic note ideas. (My initial goal has been to bring some better order to areas where I have anywhere from a dozen to four dozen notes on the same general topic and it's just too many notes to not overwhelm me when I try to figure out what all is there.)

THEN (as long as I'm still within the same chat), I could tell it:

> Based on the entire series of inputs and generated atomic note ideas in this series, can you identify the top five most important ideas? >And can you also summarize the topic in about three paragraphs? And can you summarize it further in a single paragraph?

It will do this. (So far, I've always been pretty happy with the top 5 virtually every time. Sometimes the two summaries it gives me are too similar in length and I have to ask it to try one of those again.)

And THEN, I can tell it:

>Again, based on the entire series of inputs and generated atomic note ideas, please provide an outline for a book based on this content.

This is the iffiest part of the process. Sometimes it knocks it out of the park and sometimes it's...kinda weird and needs to be regenerated. (I think it's hilarious that it often includes things like "end notes" or "acknowledgements." lol)

But overall, this gives me five different ways to look at the same batch of related content, and has helped me to make a lot of progress in identifying the structure of the notes I have.

**Reply by dvansoye** [reply]
This is amazing! I want to do this too. ❤️ Can you provide a few screenshots next time you do this so we can follow along? 🏆
